tools/picard.py

```diff
@@ -152,6 +152,8 @@ def execute(self, inBam, outFastq1, outFastq2, outFastq0=None,
         trim reads at the clipping position specified by the Illumina clipping attribute
         (which is defined by the class variable SamToFastqTool.illumina_clipping_attribute).'''
 
+        assert os.path.isfile(inBam)
+
         if tools.samtools.SamtoolsTool().isEmpty(inBam):
             # Picard SamToFastq cannot deal with an empty input BAM file
```

Comment:
The assertion checks for non-existent input, not empty input.  I added it because without it the error manifested downstream and it took me some time to figure out the cause.  In general, doing sanity-checks earlier can't hurt, no?  Unless there are so many that they obscure the core logic.